framework/src/main/java/dev/ikm/komet/framework/observable/ObservableField.java

```diff
@@ -70,6 +70,7 @@ private void handleValueChange(Object newValue) {
 
     public void writeToDataBase() {
         this.writeToDatabase(value());
+        fieldProperty.set(field().withValue(value()));
```

Comment:
Question: do we want to use field or value? Not sure but we should be mindful that there seems to be a sync issue between field and value